test/plugin/scenarios/mongodb-3.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,171 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+registryItems:
+  applications:
+    - {mongodb-3.x-scenario: nq 0}
+  instances:
+    - {mongodb-3.x-scenario: 1}
+  operationNames:
+    - mongodb-3.x-scenario: [/case/healthCheck, /case/mongodb, MongoDB/DropDatabaseOperation, MongoDB/FindOperation,
+                             MongoDB/CreateCollectionOperation, MongoDB/MixedBulkWriteOperation]
+  heartbeat: []
+
+segmentItems:
+- applicationCode: mongodb-3.x-scenario
+  segmentSize: 2
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: /case/healthCheck
+      operationId: 0
```

Comment:
health-check will be called more one when it misses a failure. So set `segmentSize` as `ge 2` and remove the segment of health-check.